src/testdir/test_functions.vim

```diff
@@ -1929,6 +1929,86 @@ func Test_readdirex()
   eval 'Xdir'->delete('rf')
 endfunc
 
+func Test_readdirex_sort()
+  CheckUnix
+  CheckNotBSD
```

Comment:
Is this check still needed? Doesn't this test work on BSDs?

Reply:
I removed.